src/build/render/components/wrapper/header.js

```diff
@@ -1,10 +1,10 @@
 const {html, DISCORD_URL} = require("../bits");
 
-const header = () => html`
+const header = (ctx, siteName) => html`
   <header class="top-header">
-    <a class="c20-logo" href="/">
+    <a class="c20-logo" href="${ctx.resolveUrl("/")}">
```

Comment:
Note that we need to resolve the URL of the homepage now because if differs based on language (e.g. "/es")